docs/sources/installation/configuration.md

```diff
@@ -81,6 +81,32 @@ export GF_AUTH_GOOGLE_CLIENT_SECRET=newS3cretKey
 export GF_PLUGIN_GRAFANA_IMAGE_RENDERER_RENDERING_IGNORE_HTTPS_ERRORS=true
 ```
 
+## Variable expansion
```

Comment:
@sakjur , have you updated [Templates and variables](https://grafana.com/docs/grafana/latest/variables/templates-and-variables/) to reflect this change?

Reply:
No, those variables are not for the configuration and these variables are not for the queries.